components/Editor/index.tsx

```diff
@@ -317,90 +318,101 @@ const Editor = ({ readOnly = false }: Props) => {
         >
           <div
             className={cn(
-              'w-full md:w-1/2 flex flex-col',
-              isThreeColumnLayout && 'md:w-1/3',
+              'w-1/2 flex flex-col',
+              isThreeColumnLayout && 'md:w-2/3',
             )}
           >
-            <div className="border-b border-gray-200 dark:border-black-500 flex items-center pl-6 pr-2 h-14 flex-none md:border-r justify-between">
-              <Header
-                codeType={codeType}
-                onCodeTypeChange={({ value }) => setCodeType(value)}
-                withLogo={isFullScreen}
-              />
-            </div>
+            <div className="flex" style={{ height: 'calc(100% - 22vh)' }}>
```

Comment:
Why are we setting the height here? Can it be removed? We should be able to just use `grow` to avoid setting any constant values here.